tutorial/markdown/nodejs/quickstart-nextjs-capella/quickstart-nextjs-capella.md

```diff
@@ -104,7 +103,7 @@ If you prefer to run this quick start using Docker, we have provided the Dockerf
 - Build the Docker image
 
 ```sh
-docker build -t docker build -t nextjs-capella-quickstart . -f Dockerfile.dev
+docker build -t nextjs-capella-quickstart . -f Dockerfile.dev
```

Comment:
I don't think this will work again.  `.` will only work with standard Dockerfile.

Reply:
<img width="1170" alt="image" src="https://github.com/couchbase-examples/couchbase-tutorials/assets/44238657/a5f58ea6-0b88-4b4c-bd9b-58e70043bee0">

Reply:
<img width="1249" alt="image" src="https://github.com/couchbase-examples/couchbase-tutorials/assets/44238657/8a81fa93-c7f0-4be1-836f-edb28d5be76a">